convert/convert/converters/coco/utils/s3_utils.py

```diff
@@ -86,13 +89,13 @@ def check_s3(self, bucket_s3: str, files_keys: List[str]) -> Any:
             self._check_files_exist(bucket_s3, files_keys)
         except FileKeyError as e:
             logger.error(f"S3 error - detail: {e}")
-            raise FileKeyError(e)
+            raise
         except BucketError as e:
             logger.error(f"S3 error - detail: {e}")
-            raise BucketError(e)
+            raise
         except urllib3.exceptions.MaxRetryError as e:
             logger.error(f"Connection error - detail: {e}")
-            raise urllib3.exceptions.MaxRetryError
+            raise
```

Comment:
I know you did not change it, but could you simplify it since you are here?
```python
 except (FileKeyError, BucketError) as e:
            logger.exception(f"S3 error - detail: {e}")
            raise
        except urllib3.exceptions.MaxRetryError as e:
            logger.exception(f"Connection error - detail: {e}")
            raise
```

Reply:
Done

Reply:
Done